api/shared/policy/list.contract.ts

```diff
@@ -1,5 +1,10 @@
 import { CampaignInterface } from './common/interfaces/CampaignInterface';
-export type ParamsInterface = void;
+
+export interface ParamsInterface {
+  territory_id?: number | null;
```

Comment:
On a vraiment la possibilité de ne pas envoyer la prop `territory_id` ou de lui passer `NULL` ?
Ce n'est pas mieux de rendre `territory_id` obligatoire ?

```suggestion
  territory_id: number | null;
```

Reply:
Quand on est côté admin on récupère actuellement toutes les campagnes.

Reply:
OK, vu aussi avec @nmrgt pour les 3 options possibles : undefined | null | number